content/software-security/what-makes-a-good-sbom.md

```diff
@@ -0,0 +1,41 @@
+type: "article"
+date: 2022-08-04T15:21:01+02:00
+lastmod: 2022-08-04T15:21:01+02:00
+contributors: ["John Speed Meyers"]
+draft: false
+images: []
+menu:
+  docs:
+    parent: "software-security"
+weight: 10
+toc: true
+---
+
+A [software bill of materials](https://edu.chainguard.dev/software-security/glossary/#sbom), or an SBOM (pronounced s-bomb), is a formal record of the components contained in a piece of software. It is analogous to an ingredients list for a recipe. And it has become recognized as one of the key building blocks of software supply chain security. Proponents rightfully point out that organizations can't secure their software if they don't know what's inside their software.
+
+As awareness and adoption of SBOM has grown, there has been a gradual ackowledgement that [not all SBOMs are created equal](https://www.chainguard.dev/unchained/not-all-sboms-are-created-equal), some are more or less useful, depending on the goals of the SBOM user and the contents of the SBOM. This guides exist to provide some guidance on evaluating the quality of an SBOM, suggesting common use cases and the data fields that support these use cases and open source SBOM quality tools.
+
+## Basic SBOM Use Cases and Reqiured SBOM Data
+
+**Identify Vulnerable Components**: SBOMs can help organizations and individuals know the unfixed vulnerabilities in their software. By providing an inventory of components, this allows a software maintainer to check whether the versions of their component are associated with any known vulnerabilities. The software maintainer can then update or patch any components with known vulnerabilities.
+
+To enable this use case, SBOMs must contain information, at a minimum, about the component name and version. Because there are many different open source software package ecosystems, it is also advantageous to include information about the package ecosystem from which a component originates. This reduces the chances of a false positive when identifying potential vulnerabilities. A [package URL](https://github.com/package-url/purl-spec) (or purl) provides this ecosystem information.
+
+**Identifying Licenses**: SBOMs can also help organizations and individuals use open source software consistent with the licensing terms of all components. By identifying all components and associated licenses, software teams can understand the legal implications of any decision related to incorporating a particular component.
+
+To enable license analysis, an SBOM must contain information about the license or licenses associated with all components.
+
+Note: There are [other SBOM use cases](https://www.atlanticcouncil.org/in-depth-research-reports/issue-brief/the-cases-for-using-sboms/), such as mapping broader ecosystem risks, but this guide currently focuses on the arguably two most well-established SBOM use cases.
+
+## Measuring SBOM Quality
+
+The tools for measuring SBOM quality--like the overall concept of SBOM quality itself--are nascent.
```

Comment:
I think we are generally doing em-dashes if that's all right. Also I would intro the next two concepts here. 

```suggestion
The tools for measuring SBOM quality — like the overall concept of SBOM quality itself — are nascent. There are currently two tools that are worth watching to evaluate SBOMs.
```

Reply:
Added in the revised commits.